src/calendar/nls/Calendar.ts

```diff
@@ -0,0 +1,13 @@
+const bundlePath = 'src/calendar/nls/common';
+const locales: string[] = [];
+
+const messages = {
+	chooseMonth: 'Choose Month',
+	chooseYear: 'Choose Year',
+	previousMonth: 'Previous Month',
+	nextMonth: 'Next Month',
+	previousYears: 'Earlier years',
+	nextYears: 'Later years'
+};
+
+export default { bundlePath, locales, messages };
```

Comment:
The most recent releases of `widget-core` (0.5.0) and `i18n` (0.4.0) change the way bundles are formatted. The `bundlePath` is no longer necessary, and `locales` is now a map of loaders:

```typescript
export default {
    locales: {
        es: () => import('./es/common') // or `require('./es/common')` where tslint dislikes `import()`
    },
    messages: { /* ... */ }
}
```

(Sorry 😖)

Reply:
Leaving this for when we update to `widget-core: 0.5.0`, as `test-extras` needs a new release first.